### ISSUE TITLE
precompute setting affects seed for Network(seed=None)

### ISSUE THREAD

**Issue by tcstewar** [nengo/nengo-loihi#58]
In nengo, if we don't specify a seed for the Network, we use np.random to generate one.  That means that this works in normal nengo:

```python
model = nengo.Network()
with model:
    ens = nengo.Ensemble(n_neurons=100, dimensions=1)
    p = nengo.Probe(ens)

np.random.seed(0)
with nengo.Simulator(model) as sim1:
    sim1.run(1.0)

np.random.seed(0)
with nengo.Simulator(model) as sim2:
    sim2.run(1.0)

assert np.allclose(sim1.data[p], sim2.data[p])
```

However, in `nengo_loihi` we get an interesting difference if we switch the precompute setting:

```python
model = nengo.Network()
with model:
    ens = nengo.Ensemble(n_neurons=100, dimensions=1)
    p = nengo.Probe(ens)

np.random.seed(0)
with nengo_loihi.Simulator(model, precompute=True) as sim1:
    sim1.run(1.0)

np.random.seed(0)
with nengo_loihi.Simulator(model, precompute=False) as sim2:
    sim2.run(1.0)

assert np.allclose(sim1.data[p], sim2.data[p])
```

```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-15-6f3ea09eef70> in <module>()
     12     sim2.run(1.0)
     13 
---> 14 assert np.allclose(sim1.data[p], sim2.data[p])

AssertionError: 
```

(Note: the `nengo_benchmarks` rely on being able to set the `np.random` seed, and this is why I was getting different outputs for `precompute=True` and `precompute=False`)

**Comment by tbekolay** [comment]
Oh, perhaps this another symptom of #29 ?

**Comment by hunse** [comment]
Addressed in #70.

**Comment by drasmuss** [comment]
#70 is now merged in
